src/runtime/audience-action-local-flows-test.js

```diff
@@ -391,6 +417,76 @@ describes.realWin('AudienceActionLocalFlow', (env) => {
         expect(eventManager.logSwgEvent).to.be.calledWith(
           AnalyticsEvent.EVENT_REWARDED_AD_GRANTED
         );
+        expect(entitlementsManager.clear).to.be.called;
+        expect(entitlementsManager.getEntitlements).to.be.called;
+        expect(env.win.localStorage.setItem).to.be.calledWith(
+          'subscribe.google.com:USER_TOKEN',
+          'xyz'
+        );
+        expect(env.win.sessionStorage.setItem).to.be.calledWith(
+          'subscribe.google.com:READ_TIME'
+        );
+      });
+
+      it('does not update entitlements when complete fails', async () => {
+        await renderAndAssertRewardedAd(
+          DEFAULT_PARAMS,
+          DEFAULT_CONFIG,
+          `{
+            "updated": false,
+            "alreadyCompleted": true,
+            "swgUserToken": "xyz"
+          }`
+        );
+
+        // Manually invoke the rewardedSlotReady callback.
+        expect(eventListeners['rewardedSlotReady']).to.not.be.null;
+        await eventListeners['rewardedSlotReady'](readyEventArg);
+        expect(eventListeners['rewardedSlotGranted']).to.not.be.null;
+        await eventListeners['rewardedSlotGranted']();
+
+        expect(env.win.fetch).to.be.calledWith(
+          'https://news.google.com/swg/_/api/v1/publication/pub1/completeaudienceaction?sut=abc&configurationId=xyz&audienceActionType=TYPE_REWARDED_AD'
+        );
+        expect(entitlementsManager.clear).to.not.be.called;
+        expect(entitlementsManager.getEntitlements).to.not.be.called;
+        expect(env.win.localStorage.setItem).to.not.be.calledWith(
+          'subscribe.google.com:USER_TOKEN',
+          'xyz'
+        );
+        expect(env.win.sessionStorage.setItem).to.not.be.calledWith(
+          'subscribe.google.com:READ_TIME'
+        );
+      });
```

Comment:
I think these can also use `.to.not.be.called` so any calls with other arguments would also fail the tests

Reply:
Done.